### PR TITLE
Fix issue 1060

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

**Comment by josenavas** [comment]
👍

File: qiita_pet/support_files/doc/source/resources.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+External services
+*****************
+
+These are external long term hosting or analytical sites that Qiita
+communicates with.
+
+European Bioinformatics Institute (EBI) - European Nucleotide Archive (ENA)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`EBI <https://www.ebi.ac.uk/ena>`__ - `ENA <https://www.ebi.ac.uk/>`__ is a long
```
Comment (tkosciol) [inline qiita_pet/support_files/doc/source/resources.rst]: EBI and ENA hyperlinks are the other way around

**Comment by mortonjt** [comment]
Should this be merged after the GNPS plugin and Humann2 plugins get in?

**Comment by antgonza** [comment]
Good question, I think we can update once those plugins are ready.

**Pull request merged**